### DISCUSSION TITLE
How to use :is() css function in Tailwind?

### DISCUSSION THREAD

**Discussion by vladbesson** [tailwindlabs/tailwindcss#12365] [Help] [answered]
For example I have this styles:

```
.my-block :is(h2) {
  color: #1f3244;
}
```

How to convert it to Tailwind?

**Comment by wongjn** [comment, marked as answer]
You could consider:

```html
<div class="my-block group">
  <h2 class="group-[.my-block]:text-[#1f3244]">
```

Or


```html
<div class="[&_:is(h2)]:text-[#1f3244]">
```

As some of many possible conversions.

**Reply by vladbesson** [reply]
Thank you!

**Reply by Octember** [reply]
Wow- had no idea this syntax exists. Super cool. Any documentation on the `&_` syntax?

**Reply by alex-krasikau** [reply]
@Octember here're the docs - https://tailwindcss.com/docs/adding-custom-styles#arbitrary-variants
